app/client/ui/FormAPI.ts

```diff
@@ -139,15 +139,16 @@ export class FormAPIImpl extends BaseAPI implements FormAPI {
     });
   }
 
-  public async createRecord(options: CreateRecordOptions): Promise<void> {
+  public async createRecord(options: CreateRecordOptions): Promise<number> {
```

Comment:
I suggest using `number|undefined` here, since that seems possible (though not normal). This will help code elsewhere to be prepared to deal with undefined.